chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/DeveloperDAOImpl.java

```diff
@@ -88,6 +89,11 @@ public DeveloperDTO create(DeveloperDTO dto) throws EntityCreationException, Ent
             }
 
             entity.setName(dto.getName());
+            if (dto.getWebsite() != null && !StringUtils.isEmpty(dto.getWebsite())) {
+                if (!ValidationUtils.isWellFormedUrl(dto.getWebsite())) {
+                    throw new EntityCreationException("Developer website is invalid");
```

Comment:
This seems business-logicky to me and I'd recommend putting it in the manager if possible. If not, I wouldn't hold up the PR for it.